reports/project-cycle-time.ts

```diff
@@ -69,8 +69,14 @@ export function process(
   issueList: IssueList,
   drillIn: (identifier: string, title: string, cards: ProjectIssue[]) => void
 ): any {
+  console.log('> cycle-time::process')
   const cycleTimeData = <CycleTimeData>{}
-  const filtered = rptLib.filterByLabel(issueList.getItems(), config['report-on-label'])
+  const reportOn = config['report-on-label']
```

Comment:
do we need to check for empty label?

Reply:
The generator applies defaults and applies user config settings if not falsey (empty string is falsey) so if the user does an empty string they should just get the default.